pkg/cmd/log.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ClusterHQ/dvol/pkg/api"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdLog(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "log",
+		Short: "Show log",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := outputLog(cmd, args, out); err != nil {
+				fmt.Fprintln(os.Stderr, err.Error())
+				os.Exit(1)
+			}
+		},
+	}
+	return cmd
+}
+
+func outputLog(cmd *cobra.Command, args []string, out io.Writer) error {
+	dvol := api.NewDvolAPI(dvolAPIOptions)
+	activeVolume, err := dvol.ActiveVolume()
+	if err != nil {
+		return err
+	}
+
+	activeBranch, err := dvol.ActiveBranch(activeVolume)
+	if err != nil {
+		return err
+	}
+
+	commits, err := dvol.ListCommits(activeVolume, activeBranch)
+	if err != nil {
+		return err
+	}
+	for i := len(commits) - 1; i >= 0; i-- {
+		commit := commits[i]
+		fmt.Fprintf(out, "commit %s\n", commit.Id)
+		fmt.Fprintln(out, "Author: Who knows <mystery@person>")
```

Comment:
I assume this is so there is a newline, but I think in the context of the rest of the code, using Fprint with a \n at the end might be less incongruous